lab-noah/test/greet-test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const greet = require('../lib/greet.js');
+const expect = require('chai').expect;
+
+describe('Greet Module', function(){
+  describe('#sayHey',function(){
+    it('should return hey noah',function(){
+      var result = greet.sayHey('noah');
+      expect(greet).to.have.property('sayHey');
+      expect(result).to.equal('hey noah');
+    });
+    it('should have a missing name error message', function(){
+      var result = greet.sayHey;
+      expect(result).to.throw(Error);
+    });
+  });
+  describe('#sayBye',function(){
+    it('should return see ya later', function(){
```

Comment:
Good idea to put a blank line above an it block so you can clearly find where the test begins and separate it from the describe blocks or other it blocks.